components/ContactCard/ContactCard.tsx

```diff
@@ -13,67 +12,96 @@ import styles from './ContactCard.module.scss';
 interface Props {
     className?: string;
     contactInfo: ContactInfo;
-    isCompact?: boolean;
+    layout: `${ContactNode.Layout}`;
     renderAvatar: ({ className }: { className: string }) => ReactNode;
+    showAvatar: boolean;
     uuid: ContactNode['uuid'];
 }
 
-function ContactCard({ className, contactInfo, isCompact = false, renderAvatar, uuid }: Props) {
-    const device = useDevice();
-    const { name, description, company, email, phone, mobile, website } = contactInfo;
+function ContactCard({ className, contactInfo, layout, renderAvatar, showAvatar, uuid }: Props) {
+    const { name, description, company, email, phone, mobile } = contactInfo;
+    const website = contactInfo.website ? new URL(contactInfo.website) : null;
     const { facebook, twitter } = getSocialHandles(contactInfo);
     const subtitle = description && company ? `${description}, ${company}` : description;
 
+    const isCard = layout === 'card';
+    const isSignature = layout === 'signature';
+
     return (
         <div
             id={`contact-${uuid}`}
             className={classNames(styles.container, className, {
-                [styles.compact]: isCompact || device.isMobile,
+                [styles.card]: isCard,
+                [styles.signature]: isSignature,
             })}
         >
             <div className={styles.content}>
-                {renderAvatar({ className: styles.avatar })}
-                <div>
+                {showAvatar && renderAvatar({ className: styles.avatar })}
+                <div className={styles.meta}>
                     <h4 className={styles.name}>{name}</h4>
                     {subtitle && <h5 className={styles.position}>{subtitle}</h5>}
                 </div>
             </div>
             <hr className={styles.divider} />
             <div className={styles.links}>
-                <div className={styles.linkGroup}>
-                    {email && (
-                        <a href={`mailto:${email}`} className={styles.link}>
-                            <IconEmail className={styles.icon} />
-                            <span className={styles.linkText}>{email}</span>
-                        </a>
-                    )}
-                    {website && (
-                        <a href={website} className={styles.link}>
+                {isCard && (
```

Comment:
I don't quite get why the links are different for two layouts 🤔

Reply:
That's a question for @iamhugofelix  I guess 🙂

Reply:
I'm not sure I got what you meant?